features/admin.tenants.v1/models/tenant.ts

```diff
@@ -48,3 +50,28 @@ export interface TenantRequestResponse {
 export interface TriggerPropTypesInterface extends IdentifiableComponentInterface {
     currentTenant: string;
 }
+
+/**
+ * Interface for the Deployment unit.
+ */
+export interface DeploymentUnit {
+    name: string;
+    displayName: string;
+    consoleHostname: string;
+}
+
+/**
+ * Interface for the response returned by the get deployment unit request.
+ */
+export interface DeploymentUnitResponse {
+    deploymentUnits: DeploymentUnit[];
+}
+
+/**
+ * Interface for the Deployment unit dropdown options.
+ */
+export interface DeploymentUnitDropdownOptionsInterface {
```

Comment:
Shall we use the DropdownChild interface itself instead of defining a new interface with same properties?